src/transaction.ts

```diff
@@ -956,6 +966,15 @@ export class Snapshot extends EventEmitter {
       .on('end', () => callback!(null, rows, stats, metadata));
   }
 
+  /**
+   * Object for Directed Read Options.
+   *
+   * @typedef {object} IDirectedReadOptions
+   * @property {google.spanner.v1.IRequestOptions} includeReplicas Contains a repeated set of
```

Comment:
Why `google.spanner.v1.IRequestOptions`?

Reply:
This was a miss. Removed this